### PR TITLE
v1.0.1

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@conda-forge-admin please pre-render

**Comment by marcelotrevisani** [comment]
@conda-grayskull show requirements

**Comment by conda-grayskull** [comment]
Working on your request...

**Comment by conda-grayskull** [comment]
================ **HOST** ================
Requirements for **host**
| Current Deps | Grayskull found |  |
|--------------|-----------------|--|
| python >=3.6 | python >=3.6 | :heavy_check_mark: |
| pip | pip | :heavy_check_mark: |
| setuptools >=41.2 | setuptools >=41.2 | :heavy_check_mark: |
| setuptools_scm |  | :x: |
| | setuptools-scm | :heavy_plus_sign: |


================ **RUN** ================
Requirements for **run**
| Current Deps | Grayskull found |  |
|--------------|-----------------|--|
| click | click | :heavy_check_mark: |
| click-default-group | click-default-group | :heavy_check_mark: |
| ensureconda >=1.3 | ensureconda >=1.3 | :heavy_check_mark: |
| jinja2 | jinja2 | :heavy_check_mark: |
| pydantic >=1.8.1 | pydantic >=1.8.1 | :heavy_check_mark: |
| python >=3.6 | python >=3.6 | :heavy_check_mark: |
| pyyaml >=5.1 | pyyaml >=5.1 | :heavy_check_mark: |
| requests >=2 | requests >=2 | :heavy_check_mark: |
| setuptools |  | :x: |
| toml | toml | :heavy_check_mark: |
| typing-extensions | typing-extensions | :heavy_check_mark: |
| | poetry | :heavy_plus_sign: |
| | ruamel.yaml | :heavy_plus_sign: |

**Comment by ocefpaf** [comment]
Poetry is optional but I believe we should ship with it for pip support. If we import pkg_resources we do need to add setuptools explicitly. Not sure about ruamel.yaml though.

**Comment by marcelotrevisani** [comment]
it seems ruamel was added recently I think, poetry is listed as required dependencies
https://github.com/conda-incubator/conda-lock/blob/71ff1a9ac1228efbc79c02f8205995babc72a190/setup.cfg#L44

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,6 +34,8 @@ requirements:
     - setuptools
     - toml
     - typing-extensions
+    - poetry
+    - ruamel.yaml
```
Comment (ocefpaf) [inline recipe/meta.yaml:38]: @mariusvniekerk what do you think about adding these two deps here? My guess is that they are both optional, right?

Comment (mariusvniekerk) [reply recipe/meta.yaml:38]: ruamel got added since conda-lock now vendors conda :(

**Review by mariusvniekerk — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (mariusvniekerk) [reply recipe/meta.yaml:38]: We should probably bound poetry to `>=1.0,<1.2`

Comment (mariusvniekerk) [reply recipe/meta.yaml:38]: > Poetry is optional but I believe we should ship with it for pip support. If we import pkg_resources we do need to add setuptools explicitly. Not sure about ruamel.yaml though.

We do import `pkg_resources`.  Wonder if we should switch over to 'importlib_metadata`

Comment (marcelotrevisani) [reply recipe/meta.yaml:38]: gotcha, gonna constrain poetry then

**Pull request merged**